### ISSUE TITLE
When the input password box pops up, do not close an open note

### ISSUE THREAD

**Issue by smilesxsy** [qing3962/password-protection#30]
Is it possible to add a setting to control not closing a currently open note
Hope it happens, thank you！！！

**Comment by qing3962** [comment]
Let me consider your suggestion.

**Comment by salttttfish** [comment]
+1，我也希望作者大大更新一下这个功能，锁定后不要关闭当前笔记。或者说解锁后可以自动重新打开之前的笔记。

**Comment by ShadiestGoat** [comment]
I agree, as an alternative to still keeping privacy - background blur

**Comment by qing3962** [comment]
Your issue had been resolved in 1.1.29 (background blur when the password verify dialog pops up):
>>> When the input password box pops up, do not close an open note.
